.travis.yml

```diff
@@ -31,7 +27,7 @@ jobs:
   include:
     # Tests the lowest set of dependencies
     - php: 7.1
-      env: deps=low SYMFONY_DEPRECATIONS_HELPER=999999
+      env: stability=lowest SYMFONY_DEPRECATIONS_HELPER=999999
```

Comment:
`lowest` is not actually a stability value. This is confusing.

Reply:
Using lowercase env variable is confusing as well.

Reply:
These are only used to label builds in the build view. The `stability` env variable is not used anywhere else.